lesson_13/edmond_sabou/home1_les13.py

```diff
@@ -0,0 +1,63 @@
+from math import sqrt
+
+
+class Perimeter:
```

Comment:
For Mixin classes, we always add the word `Mixin` as a suffix

Reply:
added sufix